pkg/apis/policy/validation/validation.go

```diff
@@ -33,8 +33,15 @@ import (
 	core "k8s.io/kubernetes/pkg/apis/core"
 	apivalidation "k8s.io/kubernetes/pkg/apis/core/validation"
 	"k8s.io/kubernetes/pkg/apis/policy"
-	"k8s.io/kubernetes/pkg/security/podsecuritypolicy/seccomp"
-	psputil "k8s.io/kubernetes/pkg/security/podsecuritypolicy/util"
+)
+
+const (
```

Comment:
these were hoisted from k8s.io/kubernetes/pkg/security/podsecuritypolicy/seccomp to allow deleting that package